docs/src/modules/utils/useCustomizationPlayground.tsx

```diff
@@ -0,0 +1,355 @@
+import * as React from 'react';
+import { useTheme, Theme } from '@mui/system';
+import pick from 'lodash/pick';
+import { blue, grey, deepPurple, amber, pink } from '@mui/material/colors';
+import { BoxProps } from '@mui/material/Box';
+import { createTheme, ThemeProvider, styled } from '@mui/material/styles';
+
+export type CustomizationLabelType = {
+  [k in 'customTheme' | 'styledComponents' | 'sxProp']: string;
+};
+
+type CustomizationItemType = {
+  type: 'warning' | 'success';
+  comments?: string;
+  componentProps?: Object;
+};
+export type CustomizationItemsType = Partial<{
+  [k in keyof CustomizationLabelType]: CustomizationItemType;
+}>;
+
+export type PickersSubcomponentType = {
+  [k: string]: { examples: CustomizationItemsType; slots: string[] };
+};
+
+export interface UseCustomizationPlaygroundProps {
+  examples: PickersSubcomponentType;
+  children?: React.ReactElement | React.ReactElement[];
+  componentName: string;
+}
+export const customizationLabels: CustomizationLabelType = {
+  customTheme: 'Custom Theme',
+  styledComponents: 'Styled Components',
+  sxProp: 'SX Prop',
+};
+
+export const DEFAULT_COLORS = { deepPurple, amber, pink, blue, grey };
```

Comment:
Do we need all these colors? Why not two, depending on the contrast needed (dark vs. light).

Reply:
There isn't a specific reason why I added all of them 🤔 
But it felt like giving more options provides a better experience.. Showing different directions that the styling could take?

Reply:
I agree that less colors = less complexity if the goal of the playground is to show how to pass CSS to some parts of the UI.